### DISCUSSION TITLE
Use GP3 as default EBS volume type ?

### DISCUSSION THREAD

**Discussion by remiflament** [bottlerocket-os/bottlerocket#4078] [Q&A] [answered]
Hello,

GP3 is cheaper than GP2 with default IOPS/Throughput 

> Pricing comparaison https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-storage-compare-volume-types.html

I saw the latest ECS AMI `bottlerocket-aws-ecs-2-x86_64-v1.20.3-5d9ac849` `ami-02d533ddabf8f304a` use GP2.

What are your thoughts on using GP3 as default in the next versions?

**Comment by cbgbt** [comment, marked as answer]
Hello, thanks for the suggestion. There's a helpful comment from @ytsssun [here](https://github.com/bottlerocket-os/bottlerocket/issues/4000#issuecomment-2140995932) which discusses Bottlerocket on `gp3` by default. We've evaluated a `gp3` root volume in the past and noted a regression in boot performance, which puts us in the position of having to make a tradeoff decision between cost and performance.

That said, boot performance on `gp3` is not necessarily static and may have since improved as the feature has been live in EC2 for some time now. We should evaluate it again.

It's worth noting too that you can always use whichever EBS volume type you prefer by specifying a [block device mapping](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/block-device-mapping-concepts.html).

**Reply by remiflament** [reply]
Thanks for this clear response! I will keep an eye on this issue.

**Comment by flaviomoringa** [comment]
Hi @cbgbt, 

I've been doing some comments on issue [4000](https://github.com/bottlerocket-os/bottlerocket/issues/4000#issuecomment-2140995932) and we've tested GP3 on the root volume and found no noticeable lost of boot performance. Although we know we could change the block device mapping (that is how we tested), unfortunately by default we are using a scaler solution that does not allow us to change block devices configurations :-(

So I would like to ask if you could again evaluate the change of the default type on the AMI to be by default GP3 for both volumes. It would decrease our EBS storage costs quite a bit.

Thanks.

**Comment by edumgui** [comment]
Hi @cbgbt,

We're willing to enable an SCP to deny GP2 usage. Nowadays, we're using Bottlerocket on Managed Node Groups and we don't want to use a custom launch template because those do not inherit the user-data from the MNG, and it will be another resource to maintain.

Maybe you can build two AMIs, keeping the default name for the GP2 and adding a '_gp3' suffix for the one with those volumes?

Thank you.
